control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -79,6 +80,8 @@ var (
 	// only once.
 	deleteDNSOperatorDeploymentOnce sync.Once
 	deleteCVORemovedResourcesOnce   sync.Once
+
+	deletionSecondsTimeout = 600 * time.Second
```

Comment:
Why is 600 seconds a meaningful threshold?

Reply:
After talking with @csrwng 10 minutes is a confortable time for a cluster to give up on HC deletion.

Reply:
can we please add a comment to the var on why 600?